### PR TITLE
Update dist_tuto.rst to include spawn mode

### PR DESCRIPTION
**Pull request by hcruiz**
Without spawn mode, few users had some issues with the first part of the tutorial.

### CODE REVIEW

**Comment by hcruiz** [comment]
@mrshenli As you suggested on the PyTorch [forum](https://discuss.pytorch.org/t/runtime-error-connection-reset-by-peer-in-init-process-group/72628/5?u=juans), I have done a PR with the addition of the `mp.set_start_method("spawn")`.

File: intermediate_source/dist_tuto.rst [hunk 1]
```diff
@@ -63,8 +63,9 @@ the following template.
     if __name__ == "__main__":
         size = 2
         processes = []
+         mp.set_start_method("spawn")
```
Comment (mrshenli) [inline intermediate_source/dist_tuto.rst:66]: wrong indent?

Comment (hcruiz) [reply intermediate_source/dist_tuto.rst]: uff, sorry, my bad. Corrected. Waiting for tests

**Review by mrshenli — APPROVED** [review]
LGTM! Thanks for contributing!

**Comment by mrshenli** [comment]
Hey @jlin27, shall we rerun tests and land this PR? Thanks!

**Pull request merged**